### ISSUE TITLE
[BUG] Update google object storage library

### ISSUE THREAD

**Issue by cin** [instaclustr/esop#66]
**Describe the bug**
A vulnerability in google's object storage library is causing Trivy to trip up and fail our CI/CD builds. Given the way this library is pulled in, I think simply building a new release will resolve this problem (since the version's not pinned in the maven pom file).  We also need to rebuild icarus with the new esop version to squash the vulnerability in icarus as well.

https://avd.aquasec.com/nvd/2020/cve-2020-7692/

```
┌──────────────────────────────────────────────────────────┬───────────────┬──────────┬───────────────────┬───────────────┬──────────────────────────────────────────────────────────────┐
│                         Library                          │ Vulnerability │ Severity │ Installed Version │ Fixed Version │                            Title                             │
├──────────────────────────────────────────────────────────┼───────────────┼──────────┼───────────────────┼───────────────┼──────────────────────────────────────────────────────────────┤
│ com.google.oauth-client:google-oauth-client (icarus.jar) │ CVE-2020-7692 │ CRITICAL │ 1.30.5            │ 1.31.0        │ google-oauth-client: missing PKCE support in accordance with │
│                                                          │               │          │                   │               │ the RFC for OAuth 2.0...                                     │
│                                                          │               │          │                   │               │ https://avd.aquasec.com/nvd/cve-2020-7692                    │
└──────────────────────────────────────────────────────────┴───────────────┴──────────┴───────────────────┴───────────────┴──────────────────────────────────────────────────────────────┘
```

**To Reproduce**
Use icarus or esop with Trivy as a vulnerability scanner.

**Expected behavior**
No vulnerabilities

**Comment by smiklosovic** [comment]
@cin do you want me to release it or you are able to scan this and verify it is ok by building it yourself from master branch?

**Comment by cin** [comment]
We haven't explicitly tested against Trivy yet, but one of my coworkers verified it's pulling in a secure version through the dependency tree output.

```
~/d/esop> mvn dependency:tree -Dincludes=com.google.oauth-client
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------< com.instaclustr:esop >------------------------
[INFO] Building instaclustr-esop 2.0.4-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ esop ---
[INFO] com.instaclustr:esop:jar:2.0.4-SNAPSHOT
[INFO] \- com.google.cloud:google-cloud-storage:jar:1.118.1:compile
[INFO]    \- com.google.oauth-client:google-oauth-client:jar:1.31.5:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.835 s
[INFO] Finished at: 2022-08-09T17:31:29+03:00
[INFO] ------------------------------------------------------------------------
```

**Comment by smiklosovic** [comment]
I ve just released 2.0.4. I guess I need to align this to Icarus as well.

**Comment by smiklosovic** [comment]
Icarus is released too.
